TLM/TLM/State/Keybinds/KeybindSetting.cs

```diff
@@ -152,6 +171,15 @@ public struct Editable {
             return ret;
         }
 
+        /// <summary>
+        /// Determines when user releases the key
+        /// </summary>
+        /// <param name="e"></param>
+        /// <returns>true once when user releases the key.</returns>
+        public bool KeyUp() {
+            return Key.IsKeyUp();
```

Comment:
I should have done the same thing for the `KeyDown()` method above

Reply:
Hmm, `KeyDown()` method above is weird, I tried to use it and after rebinding `PageUp` to something else like `[` or `comma` (theoretically not used anywhere) it stopped working, I couldn't figure out why `KeyDown()` always return `false` - I even attached debugger 😂